pkg/controller/v1beta1/inferenceservice/reconcilers/service/service_reconciler.go

```diff
@@ -98,12 +129,13 @@ func createService(componentMeta metav1.ObjectMeta, componentExt *v1beta1.Compon
 			}
 		} else {
 			port, _ := strconv.Atoi(constants.InferenceServiceDefaultHttpPort)
+			portInt32 := intstr.FromInt(port)
```

Comment:
```suggestion
			portInt32 := intstr.FromInt32(port)
```

Because [here](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/intstr#FromInt) it is noted that `FromInt` is deprecated.

Reply:
updated.